packages/ckeditor5-table/docs/features/table.md

```diff
@@ -205,6 +205,58 @@ ClassicEditor
 	.catch( ... );
 ```
 
+### Default table styles
+
+The default tables feature allows configuring the default look of tables in the editor. The configuration object should be synchronized with the {@link builds/guides/integration/content-styles editor content styles}.
```

Comment:
The beginning of this sentence seems to miss sth.

Reply:
Hm?

Reply:
```suggestion
The table styles feature allows configuring the default look of tables in the editor. The configuration object should be synchronized with the {@link builds/guides/integration/content-styles editor content styles}.
```